packages/@orchidui-vue/src/Builder/Form/OcFormBuilder.vue

```diff
@@ -1,31 +1,150 @@
 <script setup>
-// import {
-//   Button,
-//   CardInput,
-//   Checkbox,
-//   CheckboxesGroup,
-//   Criteria,
-//   Input,
-//   LinkInput,
-//   PhoneInput,
-//   Radio,
-//   RadioGroup,
-//   Select,
-//   TextArea,
-//   Toggle,
-// } from '@orchid'
+import {
+  CardInput,
+  CheckboxesGroup,
+  Input,
+  LinkInput,
+  PhoneInput,
+  RadioGroup,
+  Select,
+  TextArea,
+  Toggle,
+  RangeInput,
+  // Range Input (waiting stories)
+  // Checkbox,
+  // Criteria,
+  // Radio,
+} from "@orchid";
 
 const props = defineProps({
+  id: {
+    type: String,
+    required: true,
+  },
   jsonForm: {
     type: Array,
   },
   errors: {
     type: Object,
   },
+  values: {
+    type: Object,
+  },
 });
+
+const emit = defineEmits(["onUpdate"]);
+
+const onUpdate = (form, value, index = undefined) => {
+  emit("onUpdate", form, value, index);
+};
+
+const multipleError = (name = []) => {
+  let errorMessage = "";
+  name.forEach((fieldName) => {
+    if (props.errors[fieldName.key]) {
+      errorMessage = props.errors[fieldName.key];
+    }
+  });
+  return errorMessage;
+};
 </script>
 <template>
   <div class="form-builder">
-    <template v-for="form in props.jsonForm"> {{ form.type }} , </template>
+    <template v-for="(form, key) in jsonForm" :key="key">
+      <CardInput
+        v-if="form.type === 'CardInput'"
```

Comment:
Maybe could we use computed with returned component from switch:case by type, so that will decrease html amount to:

`<component :is="getComponentByType(form.type)" />`

Reply:
Also for `form.type` could we create some const `FormTypes` for avoid hardcode string and write just `FormTypes.Input` also we could use this exported const in form JSON when creating form

Reply:
fixed